src/objects/Hierarchy_Settings__c.object

```diff
@@ -57,6 +67,16 @@
         <type>Checkbox</type>
     </fields>
     <fields>
+        <fullName>Contacts_Addresses_Enabled__c</fullName>
```

Comment:
why does Account just have the recordtypes list, whereas contacts has both a checkbox as well as a recordtypes list?  Or did I just miss an account checkbox that already existed?